eclair-core/src/main/scala/fr/acinq/eclair/channel/InteractiveTxBuilder.scala

```diff
@@ -98,6 +98,8 @@ object InteractiveTxBuilder {
   case class RemoteFailure(cause: ChannelException) extends Failed
   // @formatter:on
 
+  case class RequireConfirmedInputs(local: Boolean, remote: Boolean)
```

Comment:
This is a bit ambiguous, in the same way that for `toSelfDelay`. Is `local` _requiring_ or _requested_ to use confirmed inputs? It's _requested_.

Maybe:
```suggestion
  case class RequireConfirmedInputs(forLocal: Boolean, forRemote: Boolean)
```
or:
```suggestion
  case class RequireConfirmedInputsFor(local: Boolean, remote: Boolean)
```